src/styles/index.ts

```diff
@@ -32,6 +32,7 @@ export const color = {
 
 export const typeface = {
   /* typefaces */
-  title: 'brandon-grotesque, Helvetica Neue, Helvetica, sans-serif',
+  title:
+    'Josefin Sans,brandon-grotesque, Helvetica Neue, Helvetica, sans-serif',
```

Comment:
@alihamoody `brandonj-grotesque` should come first - that's our typeface of choice.  Josefin is the fallback 👍

Reply:
Oops, yes of course!
Fixed :)